Sources/Testing/Running/Configuration.TestFilter.swift

```diff
@@ -126,21 +126,23 @@ extension Configuration.TestFilter {
   ///
   /// - Parameters:
   ///   - membership: How to interpret the result when predicating tests.
-  ///   - pattern: The pattern, expressed as a `Regex`-compatible regular
-  ///     expression, to match test IDs against.
+  ///   - patterns: The patterns, expressed as a `Regex`-compatible regular
+  ///     expressions, to match test IDs against.
   @available(_regexAPI, *)
-  init(membership: Membership, matching pattern: String) throws {
-    // Validate the regular expression by attempting to initialize a `Regex`
+  init(membership: Membership, matchingAnyOf patterns: some Sequence<String>) throws {
+    // Validate each regular expression by attempting to initialize a `Regex`
     // representing it, but do not preserve it. This type only represents
     // the pattern in the abstract, and is not responsible for actually
     // applying it to a test graph — that happens later during planning.
     //
     // Performing this validation here currently makes such errors easier to
     // surface when using the SwiftPM entry point. But longer-term, we should
     // make the planning phase throwing and propagate errors from there instead.
-    _ = try Regex(pattern)
+    for pattern in patterns {
+      _ = try Regex(pattern)
```

Comment:
I think at this point we can store the regexes directly.

Reply:
I can try but I think we still won't be able to due to partial API availability

Reply:
Yeah, confirmed it still cannot be stored in the `Kind` enum case due to higher API availability

Reply:
Booooo okay.